src/Microsoft.VisualStudio.Web.CodeGeneration.Utils/project.json

```diff
@@ -32,5 +33,14 @@
         "System.Threading.Tasks.Parallel": "4.0.1-*"
       }
     }
+  },
+  "packOptions": {
+    "tags": [
+      "aspnetcore",
+      "aspnetcoremvc",
```

Comment:
Can probably remove the `aspnetcoremvc` tag from the packages that aren't specific to MVC.